lib/testsuite/index.js

```diff
@@ -141,13 +142,36 @@ class TestSuite {
     this.context.modulePath = file;
   }
 
+  validateNightwatchInspectorCriteria() {
+    return (
+      this.argv.debug && 
+      Concurrency.isMasterProcess() && 
+      this.settings.desiredCapabilities.browserName === Browser.CHROME
```

Comment:
can we use `this.client.api.isChrome` here? or the better thing would be to move the logic for isChrome to utils like we did for safari and then use that everywhere.

Reply:
moved the logic for `isChrome` to utils and used that

Reply:
Can we modify this method as well to use the one in utils so that we have single source of truth, over here: 
https://github.com/nightwatchjs/nightwatch/blob/1e0668fb760008d3143b4d5cfb22f34f0497ee5e/lib/core/client.js#L98-L102

Reply:
done 👍

Reply:
also should we not check in `this.settings.capabilities` also?

Reply:
I'm not sure about this. @gravityvi thoughts?

Reply:
this has been fixed with the refactor